cloud/jenkins/psmdb_docker_build.groovy

```diff
@@ -1,27 +1,58 @@
 void build(String IMAGE_SUFFIX){
     sh """
+        set -e
+
         cd ./source/
         if [ ${IMAGE_SUFFIX} = backup ]; then
-            docker build --no-cache --progress plain --squash -t perconalab/percona-server-mongodb-operator:main-${IMAGE_SUFFIX} -f percona-backup-mongodb/Dockerfile percona-backup-mongodb
+            docker build --no-cache --progress plain --squash -t perconalab/percona-server-mongodb-operator:main-${IMAGE_SUFFIX} \
+                         -f percona-backup-mongodb/Dockerfile percona-backup-mongodb
         else
             DOCKER_FILE_PREFIX=\$(echo ${IMAGE_SUFFIX} | tr -d 'mongod')
-            docker build --no-cache --progress plain --squash -t perconalab/percona-server-mongodb-operator:main-${IMAGE_SUFFIX} -f percona-server-mongodb-\$DOCKER_FILE_PREFIX/Dockerfile percona-server-mongodb-\$DOCKER_FILE_PREFIX
-            docker build --build-arg DEBUG=1 --no-cache --progress plain --squash -t perconalab/percona-server-mongodb-operator:main-${IMAGE_SUFFIX}-debug -f percona-server-mongodb-\$DOCKER_FILE_PREFIX/Dockerfile percona-server-mongodb-\$DOCKER_FILE_PREFIX
+            docker build --no-cache --progress plain --squash -t perconalab/percona-server-mongodb-operator:main-${IMAGE_SUFFIX} \
+                         -f percona-server-mongodb-\$DOCKER_FILE_PREFIX/Dockerfile percona-server-mongodb-\$DOCKER_FILE_PREFIX
+
+            docker build --build-arg DEBUG=1 --no-cache --progress plain --squash -t perconalab/percona-server-mongodb-operator:main-${IMAGE_SUFFIX}-debug \
+                         -f percona-server-mongodb-\$DOCKER_FILE_PREFIX/Dockerfile percona-server-mongodb-\$DOCKER_FILE_PREFIX
         fi
     """
 }
 void checkImageForDocker(String IMAGE_SUFFIX){
-     withCredentials([usernamePassword(credentialsId: 'hub.docker.com', passwordVariable: 'PASS', usernameVariable: 'USER')]) {
-        sh """
-            IMAGE_SUFFIX=${IMAGE_SUFFIX}
-            IMAGE_NAME='percona-server-mongodb-operator'
-            TrivyLog="$WORKSPACE/trivy-\$IMAGE_NAME-${IMAGE_SUFFIX}-psmdb.xml"
+    try {
+             withCredentials([usernamePassword(credentialsId: 'hub.docker.com', passwordVariable: 'PASS', usernameVariable: 'USER'), string(credentialsId: 'SNYK_ID', variable: 'SNYK_ID')]) {
+                sh """
+                    IMAGE_SUFFIX=${IMAGE_SUFFIX}
+                    IMAGE_NAME='percona-server-mongodb-operator'
+                    MONGODB_VER=\$(echo ${IMAGE_SUFFIX} | tr -d '\\-debug' | tr -d 'mongod')
+                    PATH_TO_DOCKERFILE="source/percona-server-mongodb-\${MONGODB_VER}"
+                    IMAGE_TAG="main-\${IMAGE_SUFFIX}"
+                    if [ ${IMAGE_SUFFIX} = backup ]; then
+                        PATH_TO_DOCKERFILE="source/percona-backup-mongodb"
+                    elif [ ${IMAGE_SUFFIX} = operator ]; then
+                        PATH_TO_DOCKERFILE="operator-source/build"
+                        IMAGE_TAG='main'
```

Comment:
When we run release build we run it from release branch and tag is `release-1-20-0` like:
`IMAGE=perconalab/percona-server-mongodb-operator:release-1-20-0`
We need to take this into account

Reply:
fixed